plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1032,6 +1032,7 @@ blockly.block.entity_is_swimming=Is %1 swimming
 blockly.block.entity_is_tagged_in=Is %1 tagged in entity tags as %2
 blockly.block.entity_is_underwater=Is %1 under water
 blockly.block.entity_isalive=Is %1 alive
+blockly.block.entity_isaggressive=Is %1 aggressive
```

Comment:
Maybe "Is %1 currently aggressive", since this checks the current state?

Reply:
Sure, I just did it this way to follow the structure of the other procedures